### ISSUE TITLE
Notifications

### ISSUE THREAD

**Issue by jafri** [uuosio/ascdk#17]
@learnforpractice  How can we replicate:

```
[[eosio::on_notify("*::transfer")]]
[[eosio::on_notify("eosio.token::transfer")]]
```

**Comment by learnforpractice** [comment]
See https://github.com/uuosio/ascdk/tree/ascdk-dev/examples/notify

**Comment by jafri** [comment]
Closed by 1cff83138c916932bdb589849cfc1992bdf94e78
